### PR TITLE
docs: Describe how to avoid home directory persistence

### PR DESCRIPTION
**Pull request by cgwalters**
Since many if not most of our initial target use cases I think don't want this.

### CODE REVIEW

File: docs/builds.md [hunk 1]
```diff
@@ -86,6 +86,34 @@ to use the network as source of truth for authentication, using e.g. [FreeIPA](h
...
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=swap.target
+
+[Mount]
+What=tmpfs
+Where=/var/home
+Type=tmpfs
+```
```
Comment (vrothberg) [inline docs/builds.md:111]: I'd like to see an explanation of how this solves the described persistence issue.  I can interpret the tmpfs trick but assume that users who don't now the solution already may have troubles interpreting the unit.

Comment (cgwalters) [reply docs/builds.md:113]: I added a `Description` to the unit.

Comment (vrothberg) [reply docs/builds.md:113]: Can we make this more obvious?  I feel like this is the important part of this doc section.  What I am looking for explicitly is the "stating the obvious" rule from https://www.amazon.com/BUGS-Writing-Revised-Guide-Debugging/dp/020137921X.  Probably just adding that a tmpfs won't survive a reboot, so changes are non-persistent.

**Comment by lmilbaum** [comment]
@cgwalters Could you please rebase the PR and check that the Konflux pipelines are not triggered?

File: docs/builds.md (hunk 1 above)
Comment (cgwalters) [reply docs/builds.md:113]: Hmm.  I have a baseline assumption in general that the target audience for bootable containers is medium-to-advanced Linux users.  For example, we aren't going to explain things like partitions, SSH keys, packages, or systemd units.  That doesn't mean every user knows *all* of those things, but if they don't we expect them to figure out how to find out more.

`tmpfs` feels to me very much like one of those things...and if you don't know what tmpfs is, pasting it into Google right now leads to https://www.kernel.org/doc/html/latest/filesystems/tmpfs.html which explains it clearly right?

The other perhaps notable thing here is these docs are likely to get rewritten anyways for integration into downstreams.

Comment (cgwalters) [reply docs/builds.md:113]: That said, it's easy to hyperlink that, so done.

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by cgwalters** [comment]
@lmilbaum looks like the docs-ci change worked, yep.

**Pull request merged**